src/blob/index/bptree/core.rs

```diff
@@ -0,0 +1,429 @@
+/// structure of b+-tree index file from the beginning:
+/// 1. Header
+/// 2. Bloom filter
+/// 3. Sorted by keys array of record headers (serialized size of one record header is 61 bytes)
+/// 4. Metadata
+/// 5. Sorted by keys array of pairs `key - offset of record header in file` (these are leaf nodes
```

Comment:
`key - offset of record header in file` does it mean `key` and `offset of record header in file` ?

Reply:
Yes, you're right, I have chosen bad separator for pair, I change that